### DISCUSSION TITLE
Add hud textures for the factions

### DISCUSSION THREAD

**Discussion by Rampoina** [MegaGlest/megaglest-source#325] [Ideas]

**Comment by andy5995** [comment]
I know some people have complained about some HUDs in a few mods. But that doesn't mean I think a design couldn't be made that is more acceptable and better than the current UI (The icons just kind of sit on the main playing area floating in space, so to speak). There would have to be discussion on the design so it doesn't annoy people. Probably no one will be discussing this anytime soon, as development on MG is very SLOW right now. Thanks for the suggestion though, @Rampoina .
